supabase/functions/ai-execute-tool/index.ts

```
@@ -0,0 +1,1530 @@
+          
+          results.revenue_health = {
+            deposit_count: revenueDeposits.length,
+            avg_deposit_size: avgDeposit,
+            largest_deposit: largestDeposit,
+            total_revenue: totalRevenue,
+            avg_days_between_deposits: avgDaysBetweenDeposits,
+            deposit_frequency_score: avgDaysBetweenDeposits < 2 ? 5 : avgDaysBetweenDeposits < 4 ? 4 : avgDaysBetweenDeposits < 7 ? 3 : 2,
+          };
+          
+          if (type !== 'all') return;
+        }
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)